RecoMuon/L3MuonProducer/src/L3MuonCandidateProducerFromMuons.h

```diff
@@ -29,13 +29,17 @@ class L3MuonCandidateProducerFromMuons : public edm::global::EDProducer<> {
   /// destructor
   ~L3MuonCandidateProducerFromMuons() override;
 
+  /// descriptions
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
   /// produce candidates
   void produce(edm::StreamID, edm::Event&, const edm::EventSetup&) const override;
 
 private:
   // L3/GLB Collection Label
   edm::InputTag m_L3CollectionLabel;
   edm::EDGetTokenT<reco::MuonCollection> muonToken_;
+  bool m_displacedReco;
```

Comment:
```suggestion
  bool const m_displacedReco;
```
The other two could also be made `const` using the following in the ctor:
(i changed `muonToken_` to `m_muonToken` to follow the naming convention of the other data members)
```cpp
    : m_L3CollectionLabel(parameterSet.getParameter<InputTag>("InputObjects")),  // standAlone Collection Label
      m_muonToken(consumes(m_L3CollectionLabel)),
      m_displacedReco(parameterSet.getParameter<bool>("DisplacedReconstruction")) {
  LogTrace(category) << " constructor called";
  produces<RecoChargedCandidateCollection>();
```

Reply:
I have changed the name to the one you suggest and made `const` the other two attributes.